doc/components_1.md

```diff
@@ -27,16 +27,102 @@ For instance, the `mj-button` component is, on the inside, a complex HTML layout
 </table>
 ```
 
+#### Attribute lists
+
+MJML authors use _attributes_ to specify custom behavior of MJML components.
+Our documentation has _attribute lists_ for each component describing each 
+  supported attribute.
+The lists have four columns.
+The first attribute list is just below, under the ["mjml" heading](##mjml).
+
+The _attribute_ column is a complete list of attributes a component supports.
+An examples of syntax for using a attributes appears just below here
+  in discussion of the `mjml` tag and its `lang` attribute.
+
+<aside class="notice">
+In general, if MJML has an attribute of the same name as an HTML attribute,
+  the two operate similarly.
+</aside>
+
+<!--- NOTE: I understand Markdown recognizes this three-dash comment
```

Comment:
We should remove comments in doc too imo

Reply:
All instances of `<!---` and related text deleted.